lib/datadog/appsec/event.rb

```diff
@@ -142,7 +142,7 @@ def tag_and_keep!(scope, waf_result)
           scope.trace.keep! if scope.trace
 
           if scope.service_entry_span
-            scope.service_entry_span.set_tag('appsec.blocked', 'true') if waf_result.actions.include?('block')
+            scope.service_entry_span.set_tag('appsec.blocked', 'true') if waf_result.actions.key?('block_request')
             scope.service_entry_span.set_tag('appsec.event', 'true')
           end
```

Comment:
Quick question: This does match the changes done to the unit specs. Do we have integration specs that are covering this as well? E.g. if the waf library actually used `block___request`, would our tests fail?

I'm assuming the question is yes, but asking just in case.

Reply:
Yes, we have such tests in `system-tests`. No integration tests in `dd-trace-rb` though.
We aim to work on action handling soon and will address this problem also.